backend/apps/accounts/admin.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from .models import User
```

Comment:
I don't see how it can improve this line, as there is no need to change the User model, and I can hardly think of any use cases where we would need to rely on the setting here... Maybe you can tell me?

Reply:
Django has `from django.contrib.auth import get_user_model`, by the way)

Reply:
Ok, noted =)

Reply:
I'm not sure, but I thought using `settings.AUTH_USER_MODEL` or `get_user_model` would be more flexible if someday someone wants to change the user model(e.g. back to the default one), we don't need to overwrite all the code which uses concrete User model.
But now I see this could be inconvenient for our project and we can leave it the way it is now.
Just wanted to know your thoughts, thank you.